armi/nuclearDataIO/cccc.py

```diff
@@ -356,6 +356,12 @@ def rwDoubleMatrix(self, contents, *shape):
         """
         return self._rwMatrix(contents, self.rwDouble, *shape)
 
+    def rwIntMatrix(self, contents, *shape):
+        """
+        Read or write a matrix of int values.
+        """
+        return self._rwMatrix(contents, self.rwInt, *shape)
+
```

Comment:
While we're lookin at them, these could both be static methods

Reply:
My guess is there's a numpy function that could take a type and a data buffer and some size/stride info and turn it into an array. If we ever care enough about speed, might be worth thinking about using that below